### ISSUE TITLE
weird characters in metainfo.xml

### ISSUE THREAD

**Issue by umlaeute** [Yoshimi/yoshimi#192]
there's some weird characters at the beginning of the following two lines:

https://github.com/Yoshimi/yoshimi/blob/02fc43c7875aa5d601cb5c266ed27674c36627d5/desktop/metainfo/yoshimi.metainfo.xml#L38-L39

as you (don't) see, both lines start with `\u200b`  (which is a "zero-width-space" in unicode).
i can see (replacements of) the chars when opening the file in `vi`.

afaict, this makes the XML file invalid, at least this is what `appstream-util validate` tells me:

```sh
$ appstream-util validate  desktop/metainfo/yoshimi.metainfo.xml 
desktop/metainfo/yoshimi.metainfo.xml: failed to parse desktop/metainfo/yoshimi.metainfo.xml: Error on line 39 char 7: <releases> already set '
    ' and tried to replace with '
  '
$
```

**Comment by umlaeute** [comment]
the same invisible characters are also in the `yoshimi/desktop/metainfo/yoshimi.appdata.xml` file, but see #193

**Comment by abrolag** [comment]
On Thu, 10 Aug 2023 02:06:47 -0700
umläute ***@***.***> wrote:

>the same invisible characters are also in the `yoshimi/desktop/metainfo/yoshimi.appdata.xml` file, but see #193 
>

I've no idea how they got there!
Thanks for the info about appstream-util. I'd been looking for something like
this. I will be revising the metainfo file and probably get rid of appdata. Ill
also update the links, some of those are broken too :(

On a slight diversion, in the past I was able to give a title to screenshots
when uploading them to the debian project page, but that doesn't seem possible
now and they just get: "software synthesizer originally based on ZynAddSubFX2".

I've tried several different browsers.

-- 
Will J Godfrey {apparently now an 'elderly'}
https://willgodfrey.bandcamp.com/
http://yoshimi.github.io
Say you have a poem and I have a tune.
Exchange them and we can both have a poem, a tune, and a song.
